### ISSUE TITLE
css1k.com is broken

### ISSUE THREAD

**Issue by selfthinker** [jacobrask/CSS1K#148]
The website of css1k.com is broken. It looks like the code has been removed?
(This is obviously not an issue with this repository.)

**Comment by mufid** [comment]
Well, this was pretty cool project. After stumbling upon my bookmarks, i am pretty surprised this is no longer maintained. Last activity was in ~3yrs ago.

**Comment by selfthinker** [comment]
I'd be willing to fork this and provide an alternative URL/domain. (If @jacobrask doesn't mind, but the lack of answers here is pretty telling.)

Would anyone be interested in helping out with the maintenance of the repo (i.e. mostly accepting or refusing new styles)?

**Comment by jacobrask** [comment]
Feel free to fork the repository, I'll point this one to your fork. You're
welcome to use the name CSS1K.

Cheers,
Jacob
Den 8 jun 2015 09:53 skrev "Anika Henke" <notifications@github.com>:

**Comment by selfthinker** [comment]
Great, thanks!
I have just registered css1k.net. I will set everything up in the evening (BST).

**Comment by selfthinker** [comment]
http://css1k.net/ is up and running.

I removed any mentioning of the Twitter account (as a) I don't have access to it and b) I'm not a Twitter person and am not sure if I would want to keep that up-to-date). Could you make a last tweet informing people about the project's new home?

**Comment by erickramirez** [comment]
Awesome! @selfthinker Are you considering changing any of the underlying html or editing any of the rules? And are contributions still open then I take it?

**Comment by selfthinker** [comment]
Yes, contributions can now continue in my repository: https://github.com/selfthinker/CSS1K/
I decided to not make the old PRs in here a priority but I might look at them and merge a few if they are really interesting. But otherwise I will only look at PRs in my repo.

I have already changed some of the HTML but only slightly:

1. I changed all the URLs pointing to css1k.com and this repo (obviously).
2. I removed any mentioning of the Twitter and email account at the bottom.
3. I added to the intro that you can also navigate back and forth with the left and right arrow keys (which I implemented). That already broke some designs slightly, though (which I plan to fix later).

I plan to keep the HTML as intact as possible and only change things if they have minimal impact.
One thing which might be necessary is to find a way to deal differently with the many designs. I'm not sure what to do about it yet, though. I just collected some thoughts here: https://github.com/selfthinker/CSS1K/issues/2

I didn't plan to change the rules? Why would you like to edit any of them? What would you like to change?
Can you please not answer that in here but add a new issue to discuss on https://github.com/selfthinker/CSS1K/issues?
